packs/bestiary-effects/stance-hunters-precision.json

```diff
@@ -28,12 +28,26 @@
                 "key": "DamageDice",
                 "selector": "strike-damage"
             },
+            {
```

Comment:
Using `AdjustStrike` `range-increment` on NPC Strikes is not currently supported (adjust-strike.ts:87). This is here to future-proof.